hnn_core/optimization.py

```diff
@@ -0,0 +1,427 @@
+"""Parameter optimization functions."""
+
+# Authors: Blake Caldwell <blake_caldwell@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+from math import ceil, floor
+from collections import OrderedDict
+
+import numpy as np
+import scipy.stats as stats
+from scipy.optimize import fmin_cobyla
+
+from .network import Network
+from .dipole import _rmse
+
+
+def _get_range(val, multiplier):
+    range_min = max(0, val - val * multiplier / 100.)
```

Comment:
where does the 100 come from?

Reply:
honestly I don't know ... maybe @blakecaldwell can shed some light

Reply:
multiplier is a percentage, probably borrowed from when a value was read from the GUI.

Reply:
Thanks Blake! That makes sense, pretty easy to update in the docstring